freqtrade/data/converter.py

```diff
@@ -75,12 +78,17 @@ def clean_ohlcv_dataframe(data: DataFrame, timeframe: str, pair: str, *,
         logger.debug('Dropping last candle')
 
     if fill_missing:
-        return ohlcv_fill_up_missing_data(data, timeframe, pair)
+        return ohlcv_fill_up_missing_data(data, timeframe, pair, candle_type)
     else:
         return data
 
 
-def ohlcv_fill_up_missing_data(dataframe: DataFrame, timeframe: str, pair: str) -> DataFrame:
+def ohlcv_fill_up_missing_data(
+    dataframe: DataFrame,
+    timeframe: str,
+    pair: str,
+    candle_type: Optional[str] = ""
```

Comment:
Please don't put arguments to functions where it's clear that they neither need, nor use them.

That also means - change back the formatting change.